Fix all plants ferc ref

aesharpe opened the pull request:
This PR is a little miscellaneous. It's mostly in response to [PR # 1656](https://github.com/catalyst-cooperative/pudl/pull/1656) which changes the `all_plants_ferc1` table to be `plants_all_ferc1`. This PR makes sure that the rmi repo addresses this syntax switch.

It also changes a few things in the overrides notebook and module.

cmgosnell reviewed (changes requested):
This all looks minor and good! i have one question in there that is pretty minor and one request:

- can we default to always clearing the notebooks before checking them in?? it make them smaller and much more merge-able (god forbid we need to!)??

aesharpe commented on src/pudl_rmi/create_override_spreadsheets.py:
```diff
@@ -235,7 +236,7 @@ def _prep_deprish(deprish, pudl_out) -> pd.DataFrame:
     ].drop_duplicates()
     deprish.loc[:, "report_year"] = deprish.report_date.dt.year.astype("Int64")
-    deprish = deprish.merge(util_df, on=["utility_id_pudl"], how="left", validate="1:m")
+    deprish = deprish.merge(util_df, on=["utility_id_pudl"], how="left")
```
This is a good question. I removed the `validate "1:m` because it was failing, presumably because it was just 1:1 in the case of the deprish table. I think it's because we're merging on `utility_id_pudl` which may have multipe `utility_id_eia` values encompassed within it, but we're not mapping `utility_id_pudl` to `utility_id_eia`, we're mapping `utility_id_pudl` to `utility_id_pudl` so it makes sense that it's 1:1.

aesharpe commented:
I added a bb commit that changes any references to `opex_nonfuel` to `opex_total_nonfuel` so users don't confuse the nonfuel column with the other opex sub-components. I also updated the pudl repo in the sister branch to this.

aesharpe commented:
This last commit is just an add-on removing unnecessary code from dbcp and greg

cmgosnell reviewed (approved):
looks good to me!!
